feature-libs/cart/wish-list/assets/translations/en/wish-list.i18n.ts

```diff
@@ -1,3 +1,6 @@
 export const wishlist = {
   empty: 'No products in your wish list yet',
+  wishItems: {
```

Comment:
Seems we do not have a `wishList` object inside the wishList export member. Maybe this one can be wishList for general keys related to wishlist